bin/ledger2beancount

```diff
@@ -40,8 +41,11 @@ $config = $config->get;
 
 # regular expression snippets used for ledger parsing
 my $date_RE = qr{$config->{date_match}};
+my $date_RE_no_year = qr{$config->{date_match_no_year}};
```

Comment:
for consistency this should be $date_RE_no_year. At least in theory "_RE" is meant to be a variable name suffix. Same for a few other variables around here